python/TestHarness/suppressions/errors.supp

```diff
@@ -224,11 +224,10 @@
 {
    libtorch_jit
```

Comment:
Is the valgrind error coming from `jit` in BB too? If not related to jut, I would make the name of the supression a bit more general too:
```
libtorch_deregister_operator
```
isntead of 
```
libtorch_jit
```
If it's from `jit` then this is good to go.

Reply:
It is from jit yes, but I agree this rule as is now should have a more general name.